arch/arm/Makefile

```diff
@@ -14,9 +14,19 @@ else ifneq (,$(findstring v8m,$(TARGET_SUFF)))
   include arch/arm/v8m/Makefile
 else ifneq (,$(findstring v8r,$(TARGET_SUFF)))
   include arch/arm/v8r/Makefile
+else ifneq (,$(findstring v8r,$(TARGET_SUFF)))
+  include arch/arm/v8r/Makefile
+else ifeq ($(TARGET_SUBFAMILY),multilib)
+  # TODO: Try to include correct Makefile if possible in case of a multilib.
```

Comment:
I'm a little bit afraid that we will build non-fuctioning subvariants for supported targets while building the toolchain - eg. which implementation of `setjmp` will be used for `armv7m7-imx117x` with FDPIC enabled? (it's not a standard phoenix target, so it will be built only by multilib option?

Maybe we should try to inspect CFLAGS (`march/mcpu` options)? Or try using output of `$(CC) $(CFALGS)  -print-multi-directory`?

Reply:
That would be the go to solution, however, I don't think I have time to fix it

Reply:
Note, however, that when we create such a configuration and build simply libphoenix during building the target then correct implementation will be used, as install is make target is called